aiida/transports/plugins/ssh.py

```diff
@@ -361,6 +377,8 @@ def __init__(self, *args, **kwargs):
         Initialize the SshTransport class.
 
         :param machine: the machine to connect to
+        :param use_login_shell: (optional, default False)
```

Comment:
```suggestion
        :param use_login_shell: (optional, default True)
```

Reply:
I guess you don't want to change the default, right?

Reply:
Sure, a copy-paste mistake.